### PR TITLE
k8s Enhancement : Manual Update

### PR DESCRIPTION
**Pull request by pradumnnec**
The manual has been updated as per latest changes.

Updated Tutorial : https://github.com/smartfog/fogflow/blob/Manual_Update/doc/en/source/k8sIntegrationYAML.rst

### CODE REVIEW

File: doc/en/source/k8sIntegrationYAML.rst [hunk 1]
```diff
@@ -28,259 +28,147 @@ Inorder to setup the components, please refer the steps below:
...
-
-**Step 3** : Now create a namespace in kubernetes, using following command :
-
-.. code-block:: console
-
-    $kubectl create ns <user-specified> //E.g. kubectl create ns fogflow
-
-**Step 4** : Now, in order to deploy cloud components, traverse to cloud-yaml folder and configure serviceaccount.yaml file. Edit the namespace with the one created in previous step and provide a serviceaccount name as shown below.
+**Step 2** : Now, traverse to **"deployment/kubernetes/cloud-node"** folder in Fogflow repository.
+
```
Comment (naveennec) [inline doc/en/source/k8sIntegrationYAML.rst:32]: please merge both the steps

File: doc/en/source/k8sIntegrationYAML.rst [hunk 2]
```diff
@@ -28,259 +28,147 @@ Inorder to setup the components, please refer the steps below:
...
 .. code-block:: console
 
-    $kubectl create -f rabbitmq.yaml 
+    volumes:
+      - name: dgraph
+        hostPath: 
+          path: /root/kcheck/src/fogflow/deployment/kubernetes/cloud-node/dgraph      //This is to be updated as per user's own environment
 
-**Step 12** : Edit the namespace, serviceaccount value and configjson path in master.yaml as per user's environment and use below command to launch the deployments.
+**Step 5** : Now, in order to deploy cloud-node components, use below command.
```
Comment (naveennec) [inline doc/en/source/k8sIntegrationYAML.rst:62]: use below command to deploy the cloud-node component.

File: doc/en/source/k8sIntegrationYAML.rst [hunk 3]
```diff
@@ -28,259 +28,147 @@ Inorder to setup the components, please refer the steps below:
...
+    spec:
+    type: LoadBalancer
+    ports:
+        - port: 80
+        targetPort: 80
+    selector:
+        run: nginx
+    externalIPs: [172.30.48.24]  //edit this
+   
+**Step 4** : Now, edit **path** variable in dragph.yaml as per user's environment. The complete path is to be mentioned as shown below:
```
Comment (naveennec) [inline doc/en/source/k8sIntegrationYAML.rst:53]: remove now, better to write second part as : please mention the complete path as shown below:

**Review by naveennec — CHANGES_REQUESTED** [review]

File: doc/en/source/k8sIntegrationYAML.rst (hunk 1 above)
Comment (pradumnnec) [reply doc/en/source/k8sIntegrationYAML.rst]: Comment incorporated

File: doc/en/source/k8sIntegrationYAML.rst (hunk 2 above)
Comment (pradumnnec) [reply doc/en/source/k8sIntegrationYAML.rst]: Comment incorporated

File: doc/en/source/k8sIntegrationYAML.rst (hunk 3 above)
Comment (pradumnnec) [reply doc/en/source/k8sIntegrationYAML.rst]: Comment incorporated

**Review by naveennec — APPROVED** [review]

**Pull request merged**